e2e/flags.go

```diff
@@ -141,9 +169,13 @@ var GKENumNodes = flag.Int("gke-num-nodes", util.EnvInt("GKE_NUM_NODES", Default
 var GKEAutopilot = flag.Bool("gke-autopilot", util.EnvBool("GKE_AUTOPILOT", false),
 	"Whether to create GKE clusters with autopilot enabled.")
 
-// CreateClusters indicates the test framework should create GKE clusters.
+// CreateClusters indicates the test framework should create clusters.
 var CreateClusters = flag.Bool("create-clusters", util.EnvBool("CREATE_CLUSTERS", false),
-	"Whether to create GKE clusters, otherwise will assume pre-provisioned GKE clusters.")
+	"Whether to create clusters, otherwise will assume pre-provisioned clusters.")
```

Comment:
Add a "lazy" value option (non-bool) to check if the cluster exists and adopt it if it does. Then "true" should continue to error if  cluster already exists.

Reply:
done